vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,687 @@
+---
+title: "Introduction to GenomicFiles"
+authors:
+  - name: "Valerie Obenchain"
+  - name: "Michael Love"
+  - name: "Martin Morgan"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "Last modified: October 2014; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteDepends{GenomicAlignments, RNAseqData.HNRNPC.bam.chr14}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+---
+
+```{r echo=FALSE, include=FALSE}
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("BiocStyle", force = TRUE)
+```
+
+```{r include=FALSE}
+library(knitr)
+opts_chunk$set(tidy=FALSE)
+```
+
+```{r include=FALSE}
+library(BiocStyle)
+```
```

Comment:
Remove 30-33. This shouldn't be needed and lines 16-17 should include BiocStyle.